src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/WindowsRuntime/Windows/UI/ViewManagement/UISettings.cs

```diff
@@ -0,0 +1,111 @@
+using MS.Internal.PresentationCore.WindowsRuntime;
+using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Windows;
+using System.Windows.Interop;
+using System.Windows.Media;
+
+namespace MS.Internal.WindowsRuntime
+{
+    namespace Windows.UI.ViewManagement
+    {
+        internal class UISettings
+        {
+            private static readonly bool _isSupported;
+            private static object _winRtInstance;
+            private UISettingsRCW.IUISettings3 _uisettings;
+
+            static UISettings()
+            {
+                try
+                {
+                    if (GetWinRTInstance(forceInitialization: true) == null)
+                    {
+                        _isSupported = false;
+                    }
+                    _isSupported = true;
```

Comment:
this overrides the `= false` above if it does not through but returns null

Reply:
Right, it should have been before that. I will fix that.